internal/evaluator/conftest_evaluator.go

```diff
@@ -93,17 +95,65 @@ func (c conftestEvaluator) Destroy() {
 	}
 }
 
+type policyRule struct {
+	title       string
+	description string
+}
+
+type policyRules map[string]policyRule
+
+func (r *policyRules) collect(a *ast.AnnotationsRef) {
+	if a.Annotations == nil {
+		return
+	}
+
+	shortName, ok := a.Annotations.Custom["short_name"].(string)
+	if !ok {
+		return
+	}
+
+	pkg := a.GetPackage()
+	if pkg == nil {
+		return
+	}
+
+	packages := make([]string, 0, len(pkg.Path))
+	for _, p := range pkg.Path[1:] {
+		packages = append(packages, strings.Trim(p.Value.String(), `"`))
+	}
```

Comment:
There's some similar code in `internal/opa/output.go` for this also FYI, see `pathStrings` etc in `PrepTemplateFields`.